dashboard/src/components/BindingList/BindingDetails.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+
+import { IK8sApiSecretResponse, IServiceBinding } from "shared/ServiceBinding";
+import "./BindingDetails.css";
+
+interface IBindingDetailsProps {
+  binding: IServiceBinding;
+  secret?: IK8sApiSecretResponse;
+}
+
+class BindingDetails extends React.Component<IBindingDetailsProps> {
+  public render() {
+    const { binding, secret } = this.props;
+    const { instanceRef, secretName } = binding.spec;
+
+    let statuses: string[][] = [["Instance", instanceRef.name], ["Secret", secretName]];
+    if (secret) {
+      const secretData = Object.keys(secret.data).map(k => [k, atob(secret.data[k])]);
+      statuses = [...statuses, ...secretData];
+    }
+    return (
+      <dl className="BindingDetails container margin-normal">
+        {statuses.map(statusPair => {
+          const [key, value] = statusPair;
+          return [
+            <dt key={key}>{key}</dt>,
+            <dd key={value}>
+              <code>{value}</code>
```

Comment:
Unnecessary but nice to have: could we detect if value is a valid JSON and parse it prettier?

Reply:
@andresmgot I think that's a good idea, but probably not relevant to this PR? Can you create an issue for it?

Reply:
done (#403)